app/api/v2/handlers/agent_api.py

```diff
@@ -26,9 +26,13 @@ def add_routes(self, app: web.Application):
         router.add_get('/deploy_commands', self.get_deploy_commands)
         router.add_get('/deploy_commands/{ability_id}', self.get_deploy_commands_for_ability)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'],
+                          summary="Retrieve all agents",
+                          description="Retrieve all agents by criteria. Use fields from the "
```

Comment:
One last change here - this will retrieve **all** stored agents. It's a little bit confusing but the query fields (sort/include/exclude) don't actually act as criteria for selecting individual agents; they just change how the response is formatted. So for example, just including 'paw' as a query parameter in the request would only return the paw of every stored agent. The only fields accepted in the body will be in the `request_schema`, if that endpoint has one.

The /facts and /relationships endpoints are the only exceptions to this, as those accept criteria in the request body for retrieving matching objects.